rust/core-lib/assets/client_example.toml

```diff
@@ -39,3 +39,7 @@ word_wrap = false
 
 # Detect tab and newline settings on file open
 autodetect_whitespace = true
+
+# If the file doesn't end with a newline this option adds a newline to it upon saving.
```

Comment:
I would keep this short; something to the effect of, "Ensure file ends in a newline when saving".

Reply:
Ah, that does sound way better.